From 86cd0d1d89c8b4d13f06dc8a353bdd99f13c4758 Mon Sep 17 00:00:00 2001
From: Gilles Caulier <caulier.gilles@gmail.com>
Date: Wed, 18 Jan 2017 10:13:20 +0100
Subject: [PATCH 2/2] Fix compilation with clang

---
 libs/dmetadata/metaengine_p.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libs/dmetadata/metaengine_p.cpp b/libs/dmetadata/metaengine_p.cpp
index 2c83b58..2b44e06 100644
--- a/libs/dmetadata/metaengine_p.cpp
+++ b/libs/dmetadata/metaengine_p.cpp
@@ -49,7 +49,7 @@ extern "C"
 #include "digikam_debug.h"
 
 // Pragma directives to reduce warnings from Exiv2.
-#if not defined(__APPLE__) && defined(__GNUC__)
+#if !defined(__APPLE__) && defined(__GNUC__)
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #endif
@@ -723,7 +723,7 @@ void MetaEngine::Private::loadSidecarData(Exiv2::Image::AutoPtr xmpsidecar)
 }  // namespace Digikam
 
 // Restore warnings
-#if not defined(__APPLE__) && defined(__GNUC__)
+#if !defined(__APPLE__) && defined(__GNUC__)
 #pragma GCC diagnostic pop
 #endif
 
-- 
2.9.3

