--- digikam-4.0.0-beta4/doc-translated/et/CMakeLists.txt	2014-03-26 23:18:54.000000000 +0200
+++ digikam-4.0.0-beta4/doc-translated/et/CMakeLists.txt	2014-03-27 04:08:23.892847509 +0200
@@ -1 +1 @@
-kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/et/ )
+kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/et/ SUBDIR digikam )
--- digikam-4.0.0-beta4/doc-translated/it/CMakeLists.txt	2014-03-26 23:19:03.000000000 +0200
+++ digikam-4.0.0-beta4/doc-translated/it/CMakeLists.txt	2014-03-27 04:08:33.049751787 +0200
@@ -1 +1 @@
-kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/it/ )
+kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/it/ SUBDIR digikam )
--- digikam-4.0.0-beta4/doc-translated/nl/CMakeLists.txt	2014-03-26 23:19:17.000000000 +0200
+++ digikam-4.0.0-beta4/doc-translated/nl/CMakeLists.txt	2014-03-27 04:08:38.961689987 +0200
@@ -1 +1 @@
-kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/nl/ )
+kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/nl/ SUBDIR digikam )
--- digikam-4.0.0-beta4/doc-translated/pt/CMakeLists.txt	2014-03-26 23:19:25.000000000 +0200
+++ digikam-4.0.0-beta4/doc-translated/pt/CMakeLists.txt	2014-03-27 04:08:44.548631587 +0200
@@ -1 +1 @@
-kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/pt/ )
+kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/pt/ SUBDIR digikam )
--- digikam-4.0.0-beta4/doc-translated/pt_BR/CMakeLists.txt	2014-03-26 23:19:32.000000000 +0200
+++ digikam-4.0.0-beta4/doc-translated/pt_BR/CMakeLists.txt	2014-03-27 04:08:49.524579573 +0200
@@ -1 +1 @@
-kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/pt_BR/ )
+kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/pt_BR/ SUBDIR digikam )
--- digikam-4.0.0-beta4/doc-translated/sv/CMakeLists.txt	2014-03-26 23:19:41.000000000 +0200
+++ digikam-4.0.0-beta4/doc-translated/sv/CMakeLists.txt	2014-03-27 04:08:53.934533471 +0200
@@ -1 +1 @@
-kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/sv/ )
+kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/sv/ SUBDIR digikam )
--- digikam-4.0.0-beta4/doc-translated/uk/CMakeLists.txt	2014-03-26 23:19:50.000000000 +0200
+++ digikam-4.0.0-beta4/doc-translated/uk/CMakeLists.txt	2014-03-27 04:08:58.555485176 +0200
@@ -1 +1 @@
-kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/uk/ )
+kde4_create_handbook( index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/uk/ SUBDIR digikam )
