diff -up digikam-6.4.0/core/libs/facesengine/CMakeLists.txt.Wall digikam-6.4.0/core/libs/facesengine/CMakeLists.txt
--- digikam-6.4.0/core/libs/facesengine/CMakeLists.txt.Wall	2019-11-12 11:11:38.095373895 -0600
+++ digikam-6.4.0/core/libs/facesengine/CMakeLists.txt	2019-11-12 11:12:18.213526994 -0600
@@ -95,7 +95,7 @@ if(ENABLE_FACESENGINE_DNN)
         # Dlib DNN generate plenty of wrong warnings. Remove these workaround later when code will be ported to OpenCV DNN.
         REMOVE_GCC_COMPILER_WARNINGS("-Wextra")
         REMOVE_GCC_COMPILER_WARNINGS("-Wundef")
-        REMOVE_GCC_COMPILER_WARNINGS("-Wall")
+        #REMOVE_GCC_COMPILER_WARNINGS("-Wall")
 
         DISABLE_CLANG_COMPILER_WARNINGS("3.99.99" "-Wno-cast-align")
         DISABLE_CLANG_COMPILER_WARNINGS("3.99.99" "-Wno-sign-compare")
