commit fb9d7b6149e610fe38500c2438d3a6298b81cdbc
Author: Gilles Caulier <caulier.gilles@gmail.com>
Date:   Mon Feb 6 09:09:03 2012 +0100

    compile

diff --git a/digikam/sidebar.docbook b/digikam/sidebar.docbook
index a5d77c9..4d20a74 100644
--- a/digikam/sidebar.docbook
+++ b/digikam/sidebar.docbook
@@ -57,7 +57,7 @@
 
            <para>Metadata, as shown in these four tabs, can be modified and enhanced in a number of ways elsewhere: 
               <itemizedlist>
-                 <listitem><para>with the <link linkend="using-camera">camera dialog</link></para></listitem>
+                 <listitem><para>with the <ulink url="help:/digikam/using-camera.html">camera interface</ulink></para></listitem>
                  <listitem><para>the <ulink url="help:/kipi-plugins/metadataeditor.html">metadata editor</ulink></para></listitem>
                  <listitem><para>the <ulink url="help:/kipi-plugins/geolocalization.html">GPS-locator</ulink></para></listitem>
                  <listitem><para><ulink url="help:/menu-descriptions#synchronize-metadata">copying</ulink> database metadata to files</para></listitem>
@@ -379,14 +379,14 @@
                     
         <sect2 id="using-sidebargeolocation">       <title>The Geolocation Tab</title>
 
-	<para>This tab is based on the marble widget to display a rough map of orientation. You can zoom in and out with the mousewheel by either scrolling it or hold and drag it. Pan by holding down the left mouse button.</para>
+    <para>This tab is based on the marble widget to display a rough map of orientation. You can zoom in and out with the mousewheel by either scrolling it or hold and drag it. Pan by holding down the left mouse button.</para>
                 <example>   <title>The Geolocation tab</title>
                     <screenshot><screeninfo>For higher resolution maps choose "More Info"</screeninfo><mediaobject><imageobject><imagedata fileref="&path;sidebarmetadatagps.png" format="PNG" /> </imageobject></mediaobject></screenshot>
                 </example>
         <para>The left sidebar is a geo-search tool. Press the <keycombo action="press">&Ctrl;<keycap>left mouse button</keycap></keycombo> to pull-up a square as indicated. All images falling within the coordinates of that square will be shown. </para>
-	<para>On the right sidebar you will find a cross to indicate the GPS location and the selected image's name next to it. The GPS coordinates and the time information are displayed below the widget. </para>
-	<para>For higher resolution maps choose <guibutton>More Info</guibutton> button to open the detailed map service (choose before on the left scroll box) in a Internet browser.</para>
-	<para>The displayed positioning data are actually stored in the image's EXIF tags (and not yet in the database). That allows the location to be read by any other application that can understand EXIF GPS data.</para>
+    <para>On the right sidebar you will find a cross to indicate the GPS location and the selected image's name next to it. The GPS coordinates and the time information are displayed below the widget. </para>
+    <para>For higher resolution maps choose <guibutton>More Info</guibutton> button to open the detailed map service (choose before on the left scroll box) in a Internet browser.</para>
+    <para>The displayed positioning data are actually stored in the image's EXIF tags (and not yet in the database). That allows the location to be read by any other application that can understand EXIF GPS data.</para>
 
                  <sect3>                <title>What is GPS?</title>
 
@@ -398,7 +398,7 @@
                     </para>
                 </sect3>
 
-	</sect2>
+    </sect2>
 
         <sect2 id="using-sidebarcommentstags">
         <title>Comments &amp; Tags</title>
@@ -418,8 +418,8 @@
            <caution><para>
            IPTC data only supports ASCII characters and is limited to 2000 characters (old American norm). All texts will be truncated after 2000 chars, and special characters will be malformed. If you intend to use the IPTC caption field in other applications you should be compliant with these restrictions.
            </para></caution>
-	<para>After commenting, either choose the <guibutton>Apply</guibutton> button or go straight to the next image, the captions will be saved.</para>
-	<para>Next to the Apply button there is the <guibutton>More</guibutton> button. From it you can either choose to read metadata from the selected file to the database, or the other way around, to write metadata to the files (the latter take place anyway if you chose a metadata setting so that all metadata is always saved to the images).</para>
+    <para>After commenting, either choose the <guibutton>Apply</guibutton> button or go straight to the next image, the captions will be saved.</para>
+    <para>Next to the Apply button there is the <guibutton>More</guibutton> button. From it you can either choose to read metadata from the selected file to the database, or the other way around, to write metadata to the files (the latter take place anyway if you chose a metadata setting so that all metadata is always saved to the images).</para>
 
           </sect3>
 
