diff -up digikam-5.3.0/doc-translated/nl/CMakeLists.txt.doc digikam-5.3.0/doc-translated/nl/CMakeLists.txt
--- digikam-5.3.0/doc-translated/nl/CMakeLists.txt.doc	2016-11-06 04:43:01.000000000 -0600
+++ digikam-5.3.0/doc-translated/nl/CMakeLists.txt	2016-11-09 11:48:46.145716756 -0600
@@ -1,2 +1,2 @@
 KDOCTOOLS_CREATE_HANDBOOK( digikam/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/nl/ SUBDIR digikam )
-KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/nl/ SUBDIR digikam )
\ No newline at end of file
+KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/nl/ SUBDIR showfoto )
diff -up digikam-5.3.0/doc-translated/uk/CMakeLists.txt.doc digikam-5.3.0/doc-translated/uk/CMakeLists.txt
--- digikam-5.3.0/doc-translated/uk/CMakeLists.txt.doc	2016-11-06 04:43:01.000000000 -0600
+++ digikam-5.3.0/doc-translated/uk/CMakeLists.txt	2016-11-09 11:51:18.460349677 -0600
@@ -1,2 +1,2 @@
 KDOCTOOLS_CREATE_HANDBOOK( digikam/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/uk/ SUBDIR digikam )
-KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/uk/ SUBDIR digikam )
\ No newline at end of file
+KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/uk/ SUBDIR showfoto )
