diff -up digikam-1.8.0/CMakeLists.txt.libjpeg_version digikam-1.8.0/CMakeLists.txt
--- digikam-1.8.0/CMakeLists.txt.libjpeg_version	2011-01-23 13:33:57.000000000 -0600
+++ digikam-1.8.0/CMakeLists.txt	2011-02-03 13:37:16.546888612 -0600
@@ -151,17 +151,8 @@ ELSE(NOT APPLE)
     SET(CLAPACK_FOUND FALSE)
 ENDIF(NOT APPLE)
 
-# Extract version of libjpeg so that we can use the appropriate dir
-# See bug #227313, #228483
-FILE(READ "${JPEG_INCLUDE_DIR}/jpeglib.h" jpeglib_h_content)
-STRING(REGEX REPLACE ".*#define +JPEG_LIB_VERSION +([0-9]+).*" "\\1" jpeglib_version "${jpeglib_h_content}")
-MESSAGE(STATUS "Identified libjpeg version: ${jpeglib_version}")
-
-IF ("${jpeglib_version}" LESS 80)
-    SET(DIGIKAM_LIBJPEG_DIR libjpeg-62)
-ELSE ("${jpeglib_version}" LESS 80)
-    SET(DIGIKAM_LIBJPEG_DIR libjpeg-80)
-ENDIF ("${jpeglib_version}" LESS 80)
+# hardcode libjpeg version to 62
+SET(DIGIKAM_LIBJPEG_DIR libjpeg-62)
 
 # Check KDcraw version installed to compile fine.
 IF(NOT WIN32)
