diff -up digikam-5.0.0-beta3/core/libs/widgets/common/dnuminput.cpp.gcc6 digikam-5.0.0-beta3/core/libs/widgets/common/dnuminput.cpp
--- digikam-5.0.0-beta3/core/libs/widgets/common/dnuminput.cpp.gcc6	2016-01-17 17:21:32.000000000 -0600
+++ digikam-5.0.0-beta3/core/libs/widgets/common/dnuminput.cpp	2016-02-05 11:42:30.326941316 -0600
@@ -207,7 +207,7 @@ void DDoubleNumInput::setDecimals(int p)
 
 void DDoubleNumInput::setRange(double min, double max, double step)
 {
-    d->input->setRange(min, max, (int)abs(floor(log10(step))));
+    d->input->setRange(min, max, fabs(floor(log10(step))));
     d->input->setFastSliderStep(5 * step);
     d->input->setSingleStep(step);
 }
