diff -up digikam-5.0.0-beta2/core/CMakeLists.txt.MarbleQt5 digikam-5.0.0-beta2/core/CMakeLists.txt
--- digikam-5.0.0-beta2/core/CMakeLists.txt.MarbleQt5	2015-11-30 16:08:41.000000000 -0600
+++ digikam-5.0.0-beta2/core/CMakeLists.txt	2016-01-03 12:52:28.325736180 -0600
@@ -232,7 +232,7 @@ configure_file(${CMAKE_CURRENT_SOURCE_DI
 DETECT_LIBKIPI(${KIPI_MIN_VERSION})
 DETECT_LIBKSANE(${KSANE_MIN_VERSION})
 
-find_package(Marble)
+find_package(MarbleQt5)
 
 if(MARBLE_FOUND)
     # For geolocation : only KItemModels is mandatory, KBookmarks is optional in geolocation editor source code
