diff -up digikam-5.6.0/doc-translated/nl/CMakeLists.txt.doc_translated digikam-5.6.0/doc-translated/nl/CMakeLists.txt
--- digikam-5.6.0/doc-translated/nl/CMakeLists.txt.doc_translated	2017-06-13 02:48:31.000000000 -0500
+++ digikam-5.6.0/doc-translated/nl/CMakeLists.txt	2017-06-21 10:37:07.185469805 -0500
@@ -1,2 +1,2 @@
 KDOCTOOLS_CREATE_HANDBOOK( digikam/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/nl/ SUBDIR digikam )
-KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/nl/ SUBDIR digikam )
\ No newline at end of file
+KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/nl/ SUBDIR showfoto )
diff -up digikam-5.6.0/doc-translated/sv/CMakeLists.txt.doc_translated digikam-5.6.0/doc-translated/sv/CMakeLists.txt
--- digikam-5.6.0/doc-translated/sv/CMakeLists.txt.doc_translated	2017-06-13 02:48:31.000000000 -0500
+++ digikam-5.6.0/doc-translated/sv/CMakeLists.txt	2017-06-21 10:37:32.170722124 -0500
@@ -1,2 +1,2 @@
 KDOCTOOLS_CREATE_HANDBOOK( digikam/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/sv/ SUBDIR digikam )
-KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/sv/ SUBDIR digikam )
\ No newline at end of file
+KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/sv/ SUBDIR showfoto )
diff -up digikam-5.6.0/doc-translated/uk/CMakeLists.txt.doc_translated digikam-5.6.0/doc-translated/uk/CMakeLists.txt
--- digikam-5.6.0/doc-translated/uk/CMakeLists.txt.doc_translated	2017-06-13 02:48:31.000000000 -0500
+++ digikam-5.6.0/doc-translated/uk/CMakeLists.txt	2017-06-21 10:37:07.186469815 -0500
@@ -1,2 +1,2 @@
 KDOCTOOLS_CREATE_HANDBOOK( digikam/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/uk/ SUBDIR digikam )
-KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/uk/ SUBDIR digikam )
\ No newline at end of file
+KDOCTOOLS_CREATE_HANDBOOK( showfoto/index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/uk/ SUBDIR showfoto )
