diff -up digikam-2.3.0/core/libs/widgets/common/ditemtooltip.cpp.tooltips digikam-2.3.0/core/libs/widgets/common/ditemtooltip.cpp
--- digikam-2.3.0/core/libs/widgets/common/ditemtooltip.cpp.tooltips	2011-11-07 01:50:35.000000000 -0600
+++ digikam-2.3.0/core/libs/widgets/common/ditemtooltip.cpp	2011-11-22 07:24:57.108458305 -0600
@@ -65,18 +65,18 @@ DToolTipStyleSheet::DToolTipStyleSheet(c
     headEnd     = QString("</b></center></font></nobr></td></tr>");
 
     cellBeg     = QString("<tr><td><nobr><font size=\"-1\" color=\"%1\" face=\"%2\">")
-                  .arg(kapp->palette().color(QPalette::Text).name())
+                  .arg(kapp->palette().color(QPalette::ToolTipText).name())
                   .arg(font.family());
     cellMid     = QString("</font></nobr></td><td><nobr><font size=\"-1\" color=\"%1\" face=\"%2\">")
-                  .arg(kapp->palette().color(QPalette::Text).name())
+                  .arg(kapp->palette().color(QPalette::ToolTipText).name())
                   .arg(font.family());
     cellEnd     = QString("</font></nobr></td></tr>");
 
     cellSpecBeg = QString("<tr><td><nobr><font size=\"-1\" color=\"%1\" face=\"%2\">")
-                  .arg(kapp->palette().color(QPalette::Text).name())
+                  .arg(kapp->palette().color(QPalette::ToolTipText).name())
                   .arg(font.family());
     cellSpecMid = QString("</font></nobr></td><td><nobr><font size=\"-1\" color=\"%1\" face=\"%2\"><i>")
-                  .arg(kapp->palette().color(QPalette::Text).name())
+                  .arg(kapp->palette().color(QPalette::ToolTipText).name())
                   .arg(font.family());
     cellSpecEnd = QString("</i></font></nobr></td></tr>");
 }
