diff -up digikam-2.6.0/core/cmake/modules/FindLensFun.cmake.lensfun digikam-2.6.0/core/cmake/modules/FindLensFun.cmake
--- digikam-2.6.0/core/cmake/modules/FindLensFun.cmake.lensfun	2012-06-04 15:38:56.000000000 -0500
+++ digikam-2.6.0/core/cmake/modules/FindLensFun.cmake	2012-06-28 10:31:32.250644214 -0500
@@ -34,6 +34,7 @@ else (LENSFUN_LIBRARIES AND LENSFUN_INCL
       ${_lensfunIncDir}
       /opt/local/include
       /sw/include
+    PATH_SUFFIXES lensfun
   )
 
   find_library(LENSFUN_LIBRARY
