diff -urN digikam-0.8.0-orig/digikam/digikam/albumdb.h digikam-0.8.0/digikam/digikam/albumdb.h
--- digikam-0.8.0-orig/digikam/digikam/albumdb.h	2005-11-21 20:14:05.000000000 +0100
+++ digikam-0.8.0/digikam/digikam/albumdb.h	2005-12-14 00:52:52.000000000 +0100
@@ -436,7 +436,7 @@
      * @param albumID the id of the albumdb
      * @return the url of the albumdb
      */
-    QString AlbumDB::getAlbumURL(int albumID);
+    QString getAlbumURL(int albumID);
 
     /**
      * Returns the average date of all images for that album.
