--- digikam-0.7.4/digikam/sqlite/sqliteInt.h-orig	2005-08-21 05:20:42.000000000 +0200
+++ digikam-0.7.4/digikam/sqlite/sqliteInt.h	2005-09-17 15:44:54.000000000 +0200
@@ -115,7 +115,7 @@
 #   define INTPTR_TYPE int
 # else
 #   define INTPTR_TYPE long long
-#warning "WARNING: Compiling sqlite for a Big Endian Platform."
+#warning "WARNING: Compiling sqlite for a 64 bit Platform."
 #warning "WARNING: If you know this isn't true, please file a bugreport."
 # endif
 #endif
